frequency-chain/frequency-xyz
# Feat/developers section #54

# Description
Developers section, according to designs. Trying to get a single component to be style-able at the 3 different screen sizes was geting realllly messy, so I just broke up what I called the "Developer Steps" portion into 3 different components. That's also why I used a const object for text.

- Closes #54 

## Type of change

      re-approval.)

# How to Test?

1. Pull and run locally
2. Page down to "Developers" section, and also click on the link in the Nav bar.
3. Make sure you don't see any errors in console or showing up in the page.
4. Compare with design

# Checklist:
